src/algebra/group/basic.lean

```diff
@@ -401,9 +395,14 @@ by simp
 theorem sub_eq_zero : a - b = 0 ↔ a = b :=
 ⟨eq_of_sub_eq_zero, λ h, by rw [h, sub_self]⟩
 
+alias sub_eq_zero ↔ eq_of_sub_eq_zero sub_eq_zero_of_eq
```

Comment:
What does appear do when one of the lemmas already exists?

Reply:
Unfortunately, it doesn't fail.